iml-system-test-utils/src/ssh.rs

```diff
@@ -268,6 +268,17 @@ pub async fn detect_fs(host: &str) -> Result<(), CmdError> {
         .await
 }
 
+pub async fn list_fs_json(host: &str) -> Result<Vec<serde_json::Value>, CmdError> {
+    ssh_exec_cmd(host, "iml filesystem list -p json")
+        .await?
+        .checked_output()
+        .await
+        .map(|o| {
+            let v: Vec<serde_json::Value> = serde_json::from_slice(&o.stdout).unwrap();
```

Comment:
Just curious, could it also be:
```
serde_json::from_slice(&o.stdout)l.unwrap::<Vec<serde_json::Value>>()
```
?

Reply:
I tried cajoling that line to do that, but I couldn't figure it out:
```
error[E0107]: wrong number of type arguments: expected 0, found 1
   --> iml-system-test-utils/src/ssh.rs:279:61
    |
279 |         .map(|o| serde_json::from_slice(&o.stdout).unwrap::<Vec<serde_json::Value>>())
    |                                                             ^^^^^^^^^^^^^^^^^^^^^^ unexpected type argument

error: aborting due to previous error
```

Reply:
The reason that doesn't work is `unwrap` has no generic parameter to fill in.

The turbofish needs to be on `from_slice` which does have a generic parameter:

https://docs.serde.rs/serde_json/fn.from_slice.html